Gems/Atom/Tools/AtomToolsFramework/Code/Source/Viewport/ModularViewportCameraController.cpp

```diff
@@ -180,32 +181,34 @@ namespace AtomToolsFramework
                     return t * t * t * (t * (t * 6.0f - 15.0f) + 10.0f);
                 };
 
-                const float transitionT = smootherStepFn(m_animationT);
+                const auto& [transformStart, transformEnd, animationT] = m_cameraAnimation;
+
+                const float transitionT = smootherStepFn(animationT);
```

Comment:
Nit: camera/transitionT -> camera/TransitionTime? and a comment if the time is in seconds as we're doing a check against it with 1.0 below?

Reply:
Good shout, have updated 👍